tune/change-csum.c

```diff
@@ -79,10 +79,10 @@ static int check_csum_change_requreiment(struct btrfs_fs_info *fs_info, u16 new_
 		error("failed to check the dev-reaplce status: %m");
 		return ret;
 	}
-	if (ret == 0) {
-		error("running dev-replace detected, please finish or cancel it.");
```

Comment:
If ret is 0, we should check `replace_state` in the item and see if the state is acceptable:

* BTRFS_IOCTL_DEV_REPLACE_STATE_CANCELED
* BTRFS_IOCTL_DEV_REPLACE_STATE_FINISHED
* BTRFS_IOCTL_DEV_REPLACE_STATE_NEVER_STARTED (in this case I believe the dev_replace_item doesn't exist, but it's reasonable to accept this value anyway)

If the state is one of those, then there's no active replace in progress, and conversion can proceed.  Any other value for `replace_state` should error out (it's a bad value, or a new value and we don't know whether it's safe to proceed).

I would point to the tree printing code for an example of how to read a value out of an eb, but it turns out the tree printer doesn't print dev_replace items.

Would be nice to fix the spelling of "dev-reaplce" a few lines above...